storybook/cypress/e2e/modal.cy.ts

```diff
@@ -22,9 +22,11 @@ describe("Modal", () => {
       .should("have.attr", "role", "document")
       .find(".dso-close span")
       .should("have.class", "sr-only")
-      .and("have.text", "Sluiten");
-
-    // cy.percySnapshot();
+      .and("have.text", "Sluiten")
+      .then(() =>
```

Comment:
De `.then()` is niet nodig als je de argumenten uit de callback functie niet gebruikt:

```suggestion
      .should('exist');
      
    cy.matchImageSnapshot();
```